### ISSUE TITLE
about RTX GPU  and  CUDA10

### ISSUE THREAD

**Issue by deerlululu** [zju3dv/pvnet#60]
**The graphics card information**
 TITAN V  and  Tesla K20c

Which version of CUDA should be installed?
------------More Detail---------------------------------------
net663@net663:/$ nvidia-smi
Sun Aug 11 22:38:56 2019       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 430.40       Driver Version: 430.40       CUDA Version: 10.1     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  Tesla K20c          Off  | 00000000:04:00.0 Off |                    0 |
| 30%   39C    P8    19W / 225W |      0MiB /  5700MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   1  TITAN V             Off  | 00000000:81:00.0  On |                  N/A |
| 32%   46C    P8    29W / 250W |    141MiB / 12063MiB |      8%      Default |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    1      1249      G   /usr/lib/xorg/Xorg                           140MiB |
+-----------------------------------------------------------------------------+

**Comment by pengsida** [comment]
For Titan V, cuda 9 or 10 is ok.
For tesla k20c, I don't know which cuda works, since I don't have one.

**Comment by deerlululu** [comment]
NVIDIA-SMI 430.40       Driver Version: 430.40       CUDA Version: 10.1     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce GTX 1660    Off  | 00000000:01:00.0  On |                  N/A |
| 40%   38C    P8    15W / 120W |    436MiB /  5943MiB |      1%      Default 
Can this video card run this code? If you need to specify what version of cuda?

**Comment by pengsida** [comment]
Our code does not care about graphic card but cuda and pytorch version.

**Comment by pengsida** [comment]
pytorch 0.4.1 and cuda 9.0.

**Comment by deerlululu** [comment]
Thank you very much

**Comment by deerlululu** [comment]
Hello, is there a big difference between Python 3.6.8 and python 3.6.7? Would there be any problems if I used Python 3.6.8 to configure the library in requirements. txt?

**Comment by pengsida** [comment]
No.

**Comment by turinglife** [comment]
Hi @pengsida 

I have two GPUs (Titan X and Titan Xp). I use pytorch 1.1.0 and CUDA10.1.
When I trained model, sometimes I encountered a halt of loss.backward() which causes a pause of training. Do you think the version of pytorch and CUDA will lead to this issue?

I will try to use pytorch 0.4.1 and cuda 9.0 to test whether the issue still exists.

**Comment by pengsida** [comment]
Sorry, I never encounter such problem.
